.github/workflows/main_win.yaml

```diff
@@ -187,6 +230,43 @@ jobs:
           type: onhost
           os: windows
 
+  # Runs on-device integration and unit tests.
+  on-device-test:
+    needs: [initialize, build]
+    # Run ODT when on_device label is applied on PR.
+    # Also, run ODT on push and schedule if not explicitly disabled via repo vars.
+    if: |
+      fromJSON(needs.initialize.outputs.on_device_test).enabled == true && ((
```

Comment:
This line assumes that `needs.initialize.outputs.on_device_test` is an object and that enabled is a field in it. For this to be true you need to remove `.enabled` from the export in the initialize step above.

Reply:
i think i did this correctly